lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<blockquote>In practice, <strong>calculations of 3D gamma index distributions can be very time consuming; a problem that is further exacerbating in applications where multiple comparisons are required. </strong><s>one might need to perform computations of the gamma for several dose comparisons, which could result in a time consuming operation. </s></blockquote><blockquote><br></blockquote><blockquote><strong>For many applications, it is sufficient to simply calculate an accurate</strong> <s>In case we need only the final </s>pass rate.</blockquote><blockquote><br></blockquote><blockquote><strong>If we are uninterested in accurate gamma index values for gamma &gt; 1, we can suppress calculations for gamma values above 1 by passing a sutiable a value (e.g. 1.1) to the </strong><code><strong>max_gamma</strong></code><strong> parameter of the gamma function. </strong></blockquote><blockquote><strong>Moreover, random sampling of a subset of the dose distributions for gamma index calculation may also produce a sufficiently accurate pass rate. This can also introduce significant decreases in calculation time. We can take compute for a random subset by providing a suitable fraction to the </strong><code><strong>random_subset</strong></code><strong> parameter of the gamma function. </strong></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>